versions.gradle

```diff
@@ -1,8 +1,10 @@
 ext {
+    conclaveVersion='1.1'
+    conclaveRepo='<path-to-conclave-repo-folder-from-conclave-sdk>'
```

Comment:
I think what @shamsasari suggested was actually to copy the repo directory into the root of this repository and then add the relative path.